### PR TITLE
[DEPRICATED]: client code for /orders

### PR DESCRIPTION
**Pull request by jakeriksen**

### CODE REVIEW

File: src/api/orders.rs [hunk 1]
```diff
@@ -0,0 +1,110 @@
+use serde::{Deserialize, Serialize};
+
+use crate::{client::Client, error::Error};
+
+#[derive(Deserialize, Debug)]
+// The struct for placing an order - body of the post request
+pub struct OrderPlacing {
+    pub isin: String,
+    pub expires_at: Option<String>,
+    pub side: String, // "buy" or "sell"
```
Comment (Uzaaft) [inline src/api/orders.rs]: Can you change this to an enum.

File: src/api/orders.rs [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+
+impl Client {
+    /// Post and create a new order.
+    pub fn post_an_order(
+        &self,
+        _body: OrderPlacing
+    ) -> Result<OrderPlacingResponse, Error> {
+        const PATH: &str = "account/withdrawals/";
+        let resp = self.get::<OrderPlacingResponse>(PATH);
```
Comment (Uzaaft) [inline src/api/orders.rs]: Shouldn't this be using the post function instead?

Comment (jakeriksen) [reply src/api/orders.rs]: You are correct

Comment (jakeriksen) [reply src/api/orders.rs]: fixed in [7b276ff](https://github.com/Uzaaft/Septoria/pull/1/commits/7b276ffc2b41047828c576d5ff7865b281cc7ae4)

File: src/api/orders.rs (hunk 1 above)
Comment (jakeriksen) [reply src/api/orders.rs]: "side" will be changed to an enum

File: src/api/orders.rs [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+pub struct OrderPlacing {
+    pub isin: String,
+    pub expires_at: Option<String>,
+    pub side: BuyOrSell, // "buy" or "sell"
+    pub quantity: i64,
+    pub venue: Option<String>,
+}
+
+#[derive(Serialize, Deserialize, Debug)]
+pub enum BuyOrSell {
```
Comment (Uzaaft) [inline src/api/orders.rs:16]: A better name would be OrderType

Comment (jakeriksen) [reply src/api/orders.rs:16]: True

**Pull request closed without merging**